src/optimisation/Optimisation.jl

```diff
@@ -1,34 +1,35 @@
 module Optimisation
 
 using ..Turing
-using Bijectors
-using Random
-using SciMLBase: OptimizationFunction, OptimizationProblem, AbstractADType, NoAD
-
+using NamedArrays: NamedArrays
+using DynamicPPL: DynamicPPL
+import LogDensityProblems: LogDensityProblems
+import LogDensityProblemsAD: LogDensityProblemsAD
+import Optimization: Optimization
```

Comment:
```suggestion
using LogDensityProblems: LogDensityProblems
using LogDensityProblemsAD: LogDensityProblemsAD
using Optimization: Optimization
```

`import` isn't necessary if you're using qualified referneces:)

Reply:
Nice, had forgotten how these work. Done.

Is there a reason why we don't just use `import ModuleName`? Seems more concise that `using ModuleName: ModuleName`, and according to some docs I just read is equivalent. If we also wanted to make a rule out of not having mystery names in our namespaces, we could just say that the word `using` should not appear anywhere, and that would take care of it.